mailpoet/lib/Segments/DynamicSegments/FilterFactory.php

```diff
@@ -145,15 +153,25 @@ private function email($action) {
     return $this->emailAction;
   }
 
+  /**
+   * @return WooCommerceMembership
+   */
   private function wooCommerceMembership() {
```

Comment:
In cases where we can use return types in the function defintion, shouldn't we use them?
E.g. `private function wooCommerceMembership() : WooCommerceMembership {}`

Reply:
Done!